client/src/components/Footer/style.css

```diff
@@ -1,44 +1,45 @@
-.footer {
-  position:absolute;
-  left:0;
-  bottom:0;
-  right:0;
-  }
 
 .widthFormControlInFooter{
-  width: 23%;
+  padding: 5px;
   display: inline;
-  background-color: black;
+  background-color: black;  
+  width: auto;
   }
 
-.alignFooterText {
-  align-items: center;
-  /* height: 10px; */
-}
-
-/* .flex-wrapper {
-  display: flex;
-  min-height: 100vh;
-  flex-direction: column;
-  justify-content: space-between
-} */
-
 .blackFooter {
+  padding-left: 8px;
+  padding-right: 8px;
+  height: 50px;
   background-color: black;
-  border: black;
-  margin-top: 100px;
-  /* position: sticky; */
-  /* position:absolute; */
-  /* left:0;
-  bottom:0;
-  right:0; */
-  /* margin-top: auto; */
-  /* margin-top:calc(20% + 60px); */
-  /* bottom: 0; */
+  margin-top: 0px;  
 }
 
-.blackBackground {
+.blackBackground {  
+  padding-top: 0px;
+  margin-top: 0px;
   background-color: black;
-  border: black;
-  
-}
\ No newline at end of file
+  border: black;  
+}
+
+img{
+  padding: 5px;
+ }
+
+.App{
```

Comment:
We should delete this because in app.css is where styling .app should be

Reply:
working on it

Reply:
Sounds good man